### PR TITLE
Portfolio-website

### PR DESCRIPTION
**Pull request by sylvester-wamaya**
- Render projects dynamically using javascript
- Add a modal window on projects.
- Add the layout of the modal.
- Add a close button to the modal.
- Update the responsiveness of the website.

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -18,4 +18,192 @@ function close() {
...
+       <div id="mod-btn">
+       <a href="#">
+         <button type="button" class="art-btn link-btn">See live<img class="modal-icon" src="media/Icon-see live.png" alt="icon"></button>
+       <a>
+       <a href="#">
+         <button type="button" class="art-btn link-btn">See source<img class="modal-icon" src="media/Vector.png" alt="icon"></button>
+       <a>  
+        </div>
+        </div>
+        </div>
```
Comment (Strangeal) [inline app.js]: Great work so far. Kindly consider removing the `else if` section of your code as it's not a best practice to repeat yourself. Instead, you can have just the `if` section of your codebase and use `CSS media queries` to style your model to be `responsive` across all screen sizes.

File: styles.css [hunk 2]
```diff
@@ -769,4 +865,37 @@ a {
...
+    border-radius: 10px;
+    z-index: 50;
+    background-color: white;
+    width: 90vw;
+    height: auto;
+  }
+
+  #mod-btn .modal-icon {
+    margin-left: 1vw;
+  }
```
Comment (Strangeal) [inline styles.css]: Please make sure to stick to the Figma design template as much as possible. Adjust the size of your `icon` so your button text is able to align in the center to create a good user experience.
Figma:
Screenshot 2023-04-20 at 20 17 07
Your design:
Screenshot 2023-04-20 at 20 09 32

**Review by Strangeal — CHANGES_REQUESTED** [review]
Hi @sylvester-wamaya, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

Alt Text

## Highlights
Correct use of GitHub Flow ✅
Well-structured files ✅
All linter checks are passing ✅
Descriptive PR title with a great summary ✅

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by codecaiine — APPROVED** [review]
Hi @sylvester-wamaya , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- The pop up  works as expected ✔️ 
- Good UI design ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me (@codecaiine) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**